README.md

```diff
@@ -16,6 +16,22 @@ DocumentService is a library with the following functions
 - Converts an HTML document to PDF.
 - Replace placeholders in the document with actual string data.
 
+# How to setup
```

Comment:
How to setup (Windows)

Reply:
Changed heading is section to How to setup (Windows)